cmd/2019_7/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/afarbos/aoc/pkg/test"
+	"github.com/afarbos/aoc/pkg/utils"
+)
+
+func TestFindMaxThrusterSignal(t *testing.T) {
+	testData := map[int][]int{
+		43210: {3, 15, 3, 16, 1002, 16, 10, 16, 1, 16, 15, 15, 4, 15, 99, 0, 0},
+		54321: {3, 23, 3, 24, 1002, 24, 10, 24, 1002, 23, -1, 23, 101, 5, 23, 23, 1, 24, 23, 23, 4, 23, 99, 0, 0},
+		65210: {3, 31, 3, 32, 1002, 32, 10, 32, 1001, 31, -2, 31, 1007, 31, 0, 33, 1002, 33, 7, 33, 1, 33, 31,
+			31, 1, 32, 31, 31, 4, 31, 99, 0, 0, 0},
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '31', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)